pkg/acsengine/const.go

```diff
@@ -41,8 +41,8 @@ const (
 )
 
 const (
-	KubernetesHyperkubeImageName         = "hyperkube-amd64:v1.5.3"
-	KubernetesDashboardImageName         = "kubernetes-dashboard-amd64:v1.5.1"
+	KubernetesHyperkubeImageName         = "hyperkube-amd64:v1.6.0"
```

Comment:
@ritazh, Today ACS Engine does not allow user to choose K8s version (DCOS does although API will change same) but I will be working on adding that support soon. 

In the meantime it would be great if version can be parameterized like you showed above. I would prefer this format:

KubernetesHyperkubeImagePrefixName         = "hyperkube-amd64:"
KubernetesVersion = "v1.5.3"

KubernetesHyperkubeImageName can be constructed using the above. Then I will add more code later to construct it for more than one versions.

cc: @anhowe, @colemickens